app/assets/javascripts/application.js

```diff
@@ -236,8 +236,10 @@
       location.href = "?view=" + view + "&s=" + param + "&for_category=" + category;
     });
 
+    // DEPRECATED: since the introduction of PF4 and React, colorbox is being removed. Also jquery-ujs has been replaced with rails-ujs.
+    // Added #colorbox selector to target only non-React forms
     // show errors from ajax in formtastic
-    $('form').live('ajax:error', function(event, xhr, status, error) {
+    $('#colorbox form').live('ajax:error', function(event, xhr, status, error) {
```

Comment:
colorbox is also used in the Developer Portal, have you checked it doesn't affect old dev portals?
(I don't remember if this fie is loaded in dev portal)

Reply:
I'll take a more thorough look. What I'm trying to do is to prevent jquery-ujs from injecting callbacks in our new components. Maybe it's better to do it the other way around: add an excluding selector to filter our PF forms `$('form:not(.pf-c-form)')`